src/apps/menu/menu-logged-in/menu-logged-in.tsx

```diff
@@ -127,6 +119,12 @@ const MenuLoggedIn: FC = () => {
       setFeeCount(fbsFees.length);
     }
   }, [fbsFees]);
+
+  const showNotifications =
+    loansOverdue !== 0 &&
+    loansSoonOverdue !== 0 &&
+    reservationsReadyForPickup !== 0;
```

Comment:
Isn't it possible to make a falsey check instead to emphasize that this a boolean check and has nothing to do with values?
```typescript
const showNotifications =
    loansOverdue &&
    loansSoonOverdue &&
    reservationsReadyForPickup;